tests/unit/test_scmdataframe/test_pyam_compat.py

```diff
@@ -11,10 +11,9 @@ def test_to_int_value_error(test_iam_df):
     bad_val = "20311/123/1"
     idf.loc[4, "time"] = bad_val
 
-    bad_vals = idf[idf["time"] == bad_val]["time"]
     error_msg = re.escape(
         "All time values must be convertible to datetime. The following values are "
-        "not:\n{}".format(bad_vals)
+        "not:\n"
```

Comment:
Remove the last sentence?

Reply:
I've just removed it from the test because its unreasonably sensitivie if you test that the formatting is always identical. The error message still contains the bad values.